README.md

```diff
@@ -269,6 +269,21 @@ export default class App {
 }
 ```
 
+#### Decorators
+
+The `@provide` decorator lets you inject a Redux instance more succinctly:
+
+```js
+import { provide } from 'redux/react';
+
+@provide(redux)
+export default class App {
+  render() {
+    return <CounterApp />;
```

Comment:
Correct, I would assume the above is a bug. For me, it is a little confusing when missing imports. I would not understand where CounterApp came from. I feel it's better to error on the side of clarity instead of brevity.